pkg/controller/jobframework/reconciler.go

```diff
@@ -149,6 +149,16 @@ func (r *JobReconciler) ReconcileGenericJob(ctx context.Context, req ctrl.Reques
 	}
```

Comment:
if the job finishes, all pods are reclaimable. Should we update those values in the status?

Reply:
Right, I ask because of the UX. I wouldn't want to do it in a separate API call, but we can leave this as a follow up.